crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -478,21 +478,22 @@ impl GenerateStage<'_> {
   ) {
     let static_entry_chunk_reference: FxHashMap<ChunkIdx, FxHashSet<ChunkIdx>> =
       self.construct_static_entry_to_reached_dynamic_entries_map(chunk_graph);
+
+    let entry_chunk_idx =
+      chunk_graph.chunk_table.iter_enumerated().map(|(idx, _)| idx).collect::<FxHashSet<_>>();
```

Comment:
We can't use `chunk_graph.chunk_table.get(*idx)` becuase `chunk_table` is mutated during the loop.